batch/http_to_https.go

```diff
@@ -0,0 +1,86 @@
+package batch
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/spf13/viper"
+	"github.com/volatiletech/sqlboiler/boil"
+	"github.com/volatiletech/sqlboiler/queries/qm"
+
+	"github.com/Bnei-Baruch/mdb/models"
+	"github.com/Bnei-Baruch/mdb/utils"
+)
+
+type PostsHttpToHttps struct {
```

Comment:
Consider generalizing the concept a little bit to a generic string replacer.
Not necessarily http links. It could be any 2 fixed strings or maybe even regular expressions maybe